src/wallet/rpcwallet.cpp

```diff
@@ -2118,3 +2232,115 @@ Value resendwallettransactions(const Array& params, bool fHelp)
+                        "3. \"addresses\"    (string) A json array of bitcoin addresses to filter\n"
+                        "    [\n"
+                        "      \"address\"   (string) bitcoin address\n"
+                        "      ,...\n"
+                        "    ]\n"
+                        "\nResult\n"
+                        "[                   (array of json object)\n"
+                        "  {\n"
+                        "    \"txid\" : \"txid\",        (string) the transaction id \n"
+                        "    \"vout\" : n,               (numeric) the vout value\n"
+                        "    \"address\" : \"address\",  (string) the bitcoin address\n"
+                        "    \"account\" : \"account\",  (string) DEPRECATED. The associated account, or \"\" for the default account\n"
+                        "    \"scriptPubKey\" : \"key\", (string) the script key\n"
+                        "    \"amount\" : x.xxx,         (numeric) the transaction amount in btc\n"
+                        "    \"confirmations\" : n       (numeric) The number of confirmations\n"
+                        "  }\n"
+                        "  ,...\n"
+                        "]\n"
+                        
+                        "\nExamples\n"
+                        + HelpExampleCli("listunspent", "")
+                        + HelpExampleCli("listunspent", "6 9999999 \"[\\\"1PGFqEzfmQch1gKD3ra4k18PNj3tTUUSqg\\\",\\\"1LtvqCaApEdUGFkpKMM4MstjcaL4dKg8SP\\\"]\"")
+                        + HelpExampleRpc("listunspent", "6, 9999999 \"[\\\"1PGFqEzfmQch1gKD3ra4k18PNj3tTUUSqg\\\",\\\"1LtvqCaApEdUGFkpKMM4MstjcaL4dKg8SP\\\"]\"")
+                        );
+    
+    RPCTypeCheck(params, boost::assign::list_of(int_type)(int_type)(array_type));
+    
+    int nMinDepth = 1;
+    if (params.size() > 0)
+    nMinDepth = params[0].get_int();
+    
+    int nMaxDepth = 9999999;
+    if (params.size() > 1)
+    nMaxDepth = params[1].get_int();
+    
+    set<CBitcoinAddress> setAddress;
+    if (params.size() > 2) {
+        Array inputs = params[2].get_array();
+        BOOST_FOREACH(Value& input, inputs) {
+            CBitcoinAddress address(input.get_str());
+            if (!address.IsValid())
+            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, string("Invalid Bitcoin address: ")+input.get_str());
+            if (setAddress.count(address))
+            throw JSONRPCError(RPC_INVALID_PARAMETER, string("Invalid parameter, duplicated address: ")+input.get_str());
+            setAddress.insert(address);
+        }
+    }
+    
+    Array results;
+    vector<COutput> vecOutputs;
+    assert(pwalletMain != NULL);
+    LOCK2(cs_main, pwalletMain->cs_wallet);
+    pwalletMain->AvailableCoins(vecOutputs, false);
+    BOOST_FOREACH(const COutput& out, vecOutputs) {
+        if (out.nDepth < nMinDepth || out.nDepth > nMaxDepth)
+        continue;
```

Comment:
Something weird happened to the indentation of this function, for example these continues are not indented.